### PR TITLE
remove manifest creation and validation, because it used only for package name

### PR DESCRIPTION
**Pull request by Jacks0N23**

### CODE REVIEW

File: application/root-app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-        android:allowBackup="true"
-        android:extractNativeLibs="false"
-        android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+            android:name=".SampleApp"
+            android:fullBackupContent="true"
+            android:allowBackup="true"
+            android:extractNativeLibs="false"
+            android:icon="@mipmap/ic_launcher"
+            android:label="@string/app_name"
```
Comment (grine4ka) [inline application/root-app/src/main/AndroidManifest.xml:13]: Some issues with formatting, I guess.

File: plugins/android/src/main/java/tools/forma/android/feature/AndroidBinary.kt [hunk 2]
```diff
@@ -28,9 +28,9 @@ fun androidBinaryFeatureDefinition(
     featureConfiguration = featureConfiguration,
     configuration = { extension, configuration, _, formaConfiguration ->
         with(extension) {
-            compileSdkVersion(formaConfiguration.compileSdk)
+            namespace = configuration.packageName
+            compileSdk = formaConfiguration.compileSdk
 
-            defaultConfig.applicationId = configuration.packageName
```
Comment (grine4ka) [inline plugins/android/src/main/java/tools/forma/android/feature/AndroidBinary.kt:33]: Didn't know that applicationId can also be omitted. Nice! 👍🏻

File: plugins/android/src/main/java/androidProjectConfiguration.kt [hunk 3]
```diff
@@ -51,9 +47,6 @@ fun ScriptHandlerScope.androidProjectConfiguration(
             agpVersion = properties["forma.agpVersion"]!!.toString(),
             repositories = repositories,
             dataBinding = dataBinding,
-            generateMissedManifests = generateMissedManifests,
-            validateManifestPackages = validateManifestPackages,
-            javaVersionCompatibility = javaVersionCompatibility,
```
Comment (grine4ka) [inline plugins/android/src/main/java/androidProjectConfiguration.kt:56]: `javaVersionCompatibility` is also deleted. Was it on purpose or accidentally?

File: application/root-app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Jacks0N23) [reply application/root-app/src/main/AndroidManifest.xml:13]: yeah, something strange happened, thank you

File: plugins/android/src/main/java/androidProjectConfiguration.kt (hunk 3 above)
Comment (Jacks0N23) [reply plugins/android/src/main/java/androidProjectConfiguration.kt:56]: oh, it's also accidentally removing

File: plugins/android/src/main/java/tools/forma/android/feature/AndroidBinary.kt (hunk 2 above)
Comment (Jacks0N23) [reply plugins/android/src/main/java/tools/forma/android/feature/AndroidBinary.kt:33]: Yeah, here is the line from docs:

> Explicitly define the application ID. If the application ID is not explicitly defined using the applicationId property, it automatically takes on the same value as the namespace. This means that changing the namespace changes the application ID, which is usually not what you want.

[link](https://developer.android.com/studio/build/configure-app-module)

**Pull request merged**